tests/http/request.http

```diff
@@ -5,12 +5,14 @@
 # issue_number: the number of the issue, same as the one you would see in the url to that issue, e.g. "1"
 # command: the command you want to test, e.g. "/allow @UbiquityOS"
 # token: a valid GitHub token for auth, e.g. "ghs_xxxxx"
-POST http://localhost:4000/
+POST http://localhost:4001
```

Comment:
Why do you guys change the port numbers? Is it to run many plugins locally concurrently?

Reply:
Yes it was to allow multiple plugins to run locally at the same time.